packages/dds/tree/src/test/snapshots/snapshotTools.ts

```diff
@@ -10,6 +10,7 @@ import path from "path";
 import type { JsonCompatibleReadOnly } from "../../util/index.js";
 import { testSrcPath } from "../testSrcPath.cjs";
 
+// Use `npm run test:mocha:esm -- --snapshot` to set this flag.
```

Comment:
this is also available (and I feel like we prob want to recommend pnpm rather than npm):

```suggestion
// Use `pnpm run test:snapshots:regen` to set this flag.
```

Reply:
I didn't know about that. Good to know.